liboptv/tests/check_image_proc.c

```diff
@@ -141,9 +141,7 @@ END_TEST
 
 START_TEST(test_split)
 {
-    /*  A 3x3 box-blur is equivalent to lowpass_3, so that's the comparison
```

Comment:
isn't it a copy/paste of the comment from test_blur and doesn't relate to test_split ?